community/modules/remote-desktop/chrome-remote-desktop/README.md

```diff
@@ -0,0 +1,76 @@
+<!-- BEGIN_TF_DOCS -->
```

Comment:
Now we probably need more documentation here.. :)

Reply:
Added some basic documentation and added a task to improve in the future.